docs/src/content/docs/api/tools.mdx

```diff
@@ -0,0 +1,628 @@
+      }
+    }
+  };
+
+  fetch('https://api.ultravox.ai/api/tools/7f1e3b78-c71e-4b1f-9c4a-b83f7683f0d2', {
+    method: 'PUT',
+    headers: {
+      'Content-Type': 'application/json',
+      'X-API-Key': 'aBCDef.123456'
+    },
+    body: JSON.stringify(updatedToolData)
+  })
+  .then(response => response.json())
+  .then(data => console.log(data))
+  .catch(error => console.error('Error:', error));
+  ```
+  </TabItem>
+</Tabs>
+
+**Response**
+```json
+{
+  "toolId": "7f1e3b78-c71e-4b1f-9c4a-b83f7683f0d2",
+  "name": "stock_price_v2",
+  "created": "2024-09-18T14:15:22Z",
+  "definition": {
+    "description": "Get the current stock price and company info for a given symbol",
+    "dynamicParameters": [
+      {
+        "name": "symbol",
+        "location": "PARAMETER_LOCATION_QUERY",
+        "schema": {
+          "type": "string",
+          "description": "Stock symbol (e.g., AAPL for Apple Inc.)"
+        },
+        "required": true
+      }
+    ],
+    "http": {
+      "baseUrlPattern": "https://api.stockmarket.com/v2/price-and-info",
+      "httpMethod": "GET"
+    }
+  }
+}
+```
+
+## Additional Information
+
+### Tool Definition Schema
+
+
+The `definition` object in the tool creation and update requests follows the [BaseToolDefinition schema](https://api.ultravox.ai/api/schema/swagger-ui/#/). Here's a breakdown of its main components:
+
+- `description` (string): A clear, concise description of what the tool does.
+- `dynamicParameters` (array): List of parameters that can be set by the AI model when using the tool. Each parameter is an object containining:
+  - `name` (string): The name of the parameter.
+  - `location` (string): Where the parameter is used (e.g., "PARAMETER_LOCATION_QUERY", "PARAMETER_LOCATION_PATH").
+  - `schema` (object): JSON Schema definition of the parameter. This typically includes things like type, description, enum values, format, other restrictions, etc.
+  - `required` (boolean): Whether the parameter is required.
+- `staticParameters` (array, optional): List of parameters that are always set to a known, fixed value when the tool is used. These are unconditionally added when the tool is invoked. These parameters are not exposed to or set by the model. Example: you use an API for various things but want to track which requests come from your Ultravox app so you always append `utm=ultravox` to the query parameters.
+- `automaticParameters` (array, optional): Additional parameters automatically set by the system. Used when the value is not known when the tool is created but that will be known when the tool is called. Example: you want to use the unique `call_id` from ultravox as a key in your backend and you have the tool include `call_id` in the request body when your tool's API is called.
+- `requirements` (object, optional): Any specific requirements for using the tool. Currently this is used for security (e.g. API keys or HTTP Auth) and for overriding dynamic parameter values.
```

Comment:
```suggestion
- `requirements` (object, optional): Any specific requirements for using the tool. Currently this is used for security (e.g. API keys or HTTP Auth).
```

(I think required overrides takes a bit more explanation. I also don't think anyone cares yet. I mostly added this for myself for when I create a public tool for querying corpora and want to require that corpus_id be overridden when calls are started.)